app_customers/lib/util/drawer_controller.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:get/get.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+/// Drawer menu controller.
+class CustomDrawerController extends GetxController {
+  Rx<int> value = Rx<int>(0);
+  Rx<double> raduis = Rx<double>(0.0);
```

Comment:
What does radius represent??

Reply:
ist for the curves at the ends of page when she rotates